src/lib/stategraph.rs

```diff
@@ -54,11 +54,16 @@ pub fn state_exists(grm: &YaccGrammar, is: &Itemset, nt: &str, prod_off: usize,
 
     let ab_prod_off = grm.nonterm_to_prods(grm.nonterminal_off(nt)).unwrap()[prod_off];
     let ctx = &is.items[&(ab_prod_off, dot.into())];
-    for i in 0..grm.terms_len() {
+    for i in 0..grm.terms_len() + 1 {
         let bit = ctx[i];
         let mut found = false;
         for t in la.iter() {
```

Comment:
This is a helper function for tests (notice it's `#[cfg(test)]`) where it's helpful to check the context bits have the EOF symbol set (and I used "$" as a shorthand).